addon/doc/vi.md

```diff
@@ -0,0 +1,242 @@
+* "Bật các thẻ bổ sung": nếu chọn,, nó sẽ bật khả năng sử dụng [các thẻ đặc biệt](#extra-tags) trong nội dung Markdown của bạn để tự chèn những thứ như ngày giờ hiện tại. Lựa chọn này cũng có thể thiết lập trên một tài liệu cụ thể bằng cách dùng khóa các thẻ bổ sung trong [tùy chọn khối siêu dữ liệu.](#optional-metadata-block)
+* "Tạo siêu dữ liệu tương ứng từ mã nguồn HTML": khi chuyển đổi một tập tin mã nguồn HTML qua Markdown, nó sẽ nỗ lực đoán siêu dữ liệu từ mã nguồn (title - tên, lang - ngôn ngữ, filename - tên tập tin, v...v...) và tạo ra [khối siêu dữ liệu](#optionnal-metadata-block) cho bạn.
+* "Hành động mặc định trong chế độ tương tác": cho phép chọn hành động mặc định sẽ được thực hiện khi bấm phím Enter trong [Chế độ tương tác:](#interactive-mode) hiển thị nội dung đã tạo trên trình duyệt, trên màn hình ảo hay chép vào bộ nhớ tạm.
+* "Bộ công cụ Markdown": MarkdownForever cho phép bạn chọn giữa hai bộ chuyển đổi, [HTML2Text](https://pypi.org/project/html2text/) và [HTML2Markdown.](https://pypi.org/project/html2markdown/) Chỉ việc thử nghiệm và chọn cái nào bạn thích, căn cứ trên nhu cầu của bạn hoặc kết quả cho ra.
+* "Markdown2 bổ sung": xem <https://github.com/trentm/python-markdown2/wiki/Extras>.
+* "Đường dẫn": tại đây bạn có thể thiết lập nơi lưu mặc định trên ổ đĩa cho các tài liệu đã chuyển đổi. Nó có ích nếu bạn luôn dùng một thư mục để lưu tất cả công việc của mình.
+* "Quản lý mẫu HTML": sẽ mở một hộp thoại cho phép thêm, chỉnh sửa và xóa các mẫu HTML. Với mẫu, bạn sẽ có thể tùy chỉnh về mặt hình thức của tài liệu HTML bằng CSS (cascading styles sheets), ngôn ngữ đáng tin cậy cho kiểu dáng trang. Nó cho phép bạn thay đổi màu, bố cục hoặc thêm hình ảnh vào sản phẩm của bạn và tạo mẫu cho mọi nhu cầu cụ thể. Tất nhiên, có nhiều hướng dẫn học CSS trên web và [cái này](https://www.htmldog.com/guides/css/beginner/) có thể là tài liệu tốt để bắt đầu.
+
+## Tổng hợp phím lệnh
+
+- *NVDA+CTRL+i*: chế độ tương tác.
+- *NVDA+ALT+b*: chuyển đổi Markdown thành HTML. Kết quả hiển thị trên trình duyệt mặc định.
+- *NVDA+ALT+n*: chuyển đổi Markdown thành HTML. Kết quả hiển thị trên màn hình ảo của NVDA.
+- *NVDA+ALT+k*: chuyển đổi HTML thành markdown. Kết quả hiển thị trên màn hình ảo của NVDA.
+- *NVDA+ALT+l*: chuyển đổi Markdown thành mã nguồn HTML. Kết quả hiển thị trên màn hình ảo của NVDA.
+- *NVDA+SHIFT+g*: chuyển đổi HTML thành Markdown. Kết quả được chép vào bộ nhớ tạm.
+- *NVDA+SHIFT+h*: chuyển đổi Markdown thành định dạng HTML . kết quả được chép vào bộ nhớ tạm.
+- *NVDA+CTRL+h*: chuyển đổi Markdown thành mã nguồn HTML. Kết quả được chép vào bộ nhớ tạm.
+
+## Thông tin Add-on
+
+* Tác giả:
```

Comment:
Please add an entry for Vietnamese here. Also feel free to update the English documentation for this.

Reply:
Done, and I hope I understand you clearly. I also fix some spelling error that I see.